src/psyclone/transformations.py

```diff
@@ -1581,16 +1698,18 @@ def name(self):
         ''' Returns the name of this transformation as a string.'''
         return "Dynamo0p3LoopColourTrans"
 
-    def apply(self, node):
+    def apply(self, node, options=None):
         '''Performs Dynamo0.3-specific error checking and then uses the parent
         class to convert the Loop represented by :py:obj:`node` into a
         nested loop where the outer loop is over colours and the inner
         loop is over cells of that colour.
 
         :param node: the loop to transform.
         :type node: :py:class:`psyclone.dynamo0p3.DynLoop`
+        :param node: the loop to transform.
```

Comment:
Cut-n-paste error :-) You need to document `options`...

Reply:
Oops.